test/functional/eval/timer_spec.lua

```diff
@@ -222,12 +229,17 @@ describe('timers', function()
     source([[
       let g:val = 0
       func! MyHandler(timer)
+        while !g:val
+          return
```

Comment:
adding a small sleep (1-10ms) is usually a good idea, to avoid a hot loop.

Reply:
Given through the timer delay (100ms), no?